kivy/core/window/_window_sdl2.pyx

```diff
@@ -652,10 +652,7 @@ cdef class _WindowSDL2Storage:
             pass
 
     def flip(self):
-        win = self.win
-        with nogil:
-            SDL_GL_SwapWindow(win)
-            cgl.glFinish()
+        SDL_GL_SwapWindow(self.win)
```

Comment:
I'm actually curious to know which change/line specifically made the regression and why